nix/overlay.nix

```diff
@@ -190,4 +191,16 @@ in
       hash = "sha256-75QrtuOduUNT9g2RJRWUow8ESBqsDDXCMGVNQKFc+SE=";
     };
   };
+
+  vast-ui = nnbp.buildYarnPackage {
+    name = "vast-ui";
+    src = ../plugins/web/ui; # TODO use nix-filter ?
```

Comment:
RE: Not sure what the best approach is here, I think we basically want to exclude any build artifacts here, if possible without repeating the contents of `.gitignore`.